src/table/context/TableContext.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useReducer, createContext } from 'react';
+import PropTypes from 'prop-types';
+import { createSelectorProvider } from './createSelectorProvider';
+import { reducer } from '../utils/selections-utils';
+
+export const TableContext = createContext();
+
+const ProviderWithSelector = createSelectorProvider(TableContext);
+
+export const TableContextProvider = ({ children, selectionsAPI, cellCoordMock, selectionDispatchMock }) => {
```

Comment:
why cellCoordMock and selectionDispatchMock are needed?

Reply:
for testing, need to spy on the selectionDispatch and need to set focusedCellCoord to a different value for a test in the header

Reply:
oki